css/style.css

```
@@ -0,0 +1,243 @@
+*,*:before,*:after {
+    box-sizing: border-box;
+}
+
+/*Header start*/
+
+.ikit-logo {
+    width: 200px;
+}
+
+.avatar {
+    background-color: #ffffff;
+    border-radius: 16px;
+    margin: 4px;
+}
+
+.profile {
+    margin: 4px;
+    font-weight: 400;
+    font-size: 15px;
+}
+
+.menu {
+    background-color: #1F2937;
+    color: #ffffff;
+}
+
+.menu__content {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu__leading-content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu__trailing-content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.link {
+    display: flex;
+    flex-direction: row;
+    flex-flow: nowrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.nav__link{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0px 10px 0px 0px;
+    padding: 8px 12px;
+}
+
+.nav__link a{
+    color: #D1D5DB;
+    text-decoration: none;
+    font-weight: 500;
+}
+
+.nav__link.active {
+    background-color: #111827;
+    border-radius: 6px;
+}
```

так не делаем, все по бему, не исп. межблочный каскад

```suggestion
// если исп. мои правки в html
.nav__link{
    color: #D1D5DB;
    text-decoration: none;
    font-weight: 500;
}

.nav__link--active {
    background-color: #111827;
    border-radius: 6px;
}
```